src/cli.rs

```diff
@@ -45,11 +45,12 @@ pub(crate) fn parse_startup_args() -> Result<LdkUserInfo, ()> {
 		return Err(());
 	}
 	let bitcoind_rpc_info = env::args().skip(1).next().unwrap();
-	let bitcoind_rpc_info_parts: Vec<&str> = bitcoind_rpc_info.split("@").collect();
+	let mut bitcoind_rpc_info_parts: Vec<&str> = bitcoind_rpc_info.rsplitn(2,"@").collect();
```

Comment:
Here and on the line below, `rustfmt` demands a space after the `,` in the parameter list.

Reply:
didn't know about this tool yet, thanks, should be fixed in the latest commit